### ISSUE TITLE
Save a playlist with a relative path

### ISSUE THREAD

**Issue by lore7979** [vzhd1701/gridplayer#109]
Hi, I'd like to know if it is possible to save a playlist using a relative path for the video files.
At the momento the videos are saved with a full path. Ex:

C:\WTT\2022\000deg-P30\AXIS-P3245-LV_500.mp4

I'd like save a playlist in order to be opened on different PCs, something like:

..\VIDEO\AXIS-P3245-LV_500.mp4

Is it possible?
thanks Lorenzo

**Comment by sum1game** [comment]
This feature is very useful, as it allows for convenient switching between computers at home and in the office. Very good.

**Comment by nazarux** [comment]
> This feature is very useful, as it allows for convenient switching between computers at home and in the office. Very good.

I agree! ...and many other reasons - organizing your media is just another one.
